pages/linux/smbmap.md

```diff
@@ -0,0 +1,36 @@
+# smbmap
+
+> SMB enumeration tool.
+> More information: <https://github.com/ShawnDEvans/smbmap>.
+
+- Enumerate SMB shares and permissions on a host:
```

Comment:
I prefer the verb display, if this command print information in the terminal:
```suggestion
- Display SMB shares and permissions on a host:
```
The same for the others examples.

Reply:
OK, adapted.